website/src/components/visualisation/Resources/LineGraph.tsx

```diff
@@ -0,0 +1,147 @@
+import React from "react";
+import { Brush, ResponsiveContainer, LineChart, Line, YAxis, XAxis, CartesianGrid, Tooltip, Legend, Surface, Symbols, ReferenceLine } from '../../../../node_modules/recharts';
```

Comment:
```suggestion
import { Brush, ResponsiveContainer, LineChart, Line, YAxis, XAxis, CartesianGrid, Tooltip, Legend, Surface, Symbols, ReferenceLine } from 'recharts';
```
don't need to specify the direct path if it's a node module :)